vnext/Desktop/WebSocketResourceFactory.cpp

```diff
@@ -15,11 +15,10 @@ namespace Microsoft::React {
 #pragma region IWebSocketResource static members
 
 /*static*/
-shared_ptr<IWebSocketResource>
-IWebSocketResource::Make(const string &urlString, bool legacyImplementation, bool acceptSelfSigned) {
-  if (GetFeatureGate("UseWinRTWebSocket")) {
+shared_ptr<IWebSocketResource> IWebSocketResource::Make(const string &urlString) {
+  if (!GetFeatureGate("UseBeastWebSocket")) {
```

Comment:
>UseBeastWebSocket [](start = 23, length = 17)

Just checking this is not supposed to be `UseBest` or `UseBoost`

Reply:
Answering myself :) Yes the library is called `Beast` :)

---
In reply to: [465178194](https://github.com/microsoft/react-native-windows/pull/5629#discussion_r465178194) [](ancestors = 465178194)